### PR TITLE
Add econstor to harvesters

### PR DESCRIPTION
**Pull request by zamattiac**
- OAI, so no normalizer
- URL from OAI list

### CODE REVIEW

**Comment by zamattiac** [comment]
Example record OTW

File: providers/eu/econstor/__init__.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <dc:title>Better schools for Europe</dc:title>
+        <dc:type>doc-type:article</dc:type>
+        <dc:identifier>ppn:334826128</dc:identifier>
+        <dc:rights>http://www.econstor.eu/dspace/Nutzungsbedingungen</dc:rights>
+        <dc:identifier>RePEc:zbw:ifwkie:2639</dc:identifier>
+    </oai_dc:dc>
+</metadata>
+</record>
+
+"""
```
Comment (chrisseto) [inline providers/eu/econstor/__init__.py]: Add a newline to the end of this file

**Comment by zamattiac** [comment]
@chrisseto done and done

**Comment by zamattiac** [comment]
@chrisseto conflicts resolved

**Pull request merged**